DESCRIPTION

```diff
@@ -0,0 +1,18 @@
+Package: vitalsigns
+Title: A Utility Package for Conservation International's Vital Signs Project 
+Version: 0.1
+Authors@R: person("Thomas", "Johnson", email = "thomascjohnson@gmail.com",
+                  role = c("aut", "cre"))
+Description: A package that interfaces Google Drive and other services to
```

Comment:
Delete "A package that" per CRAN requirements. Should read "Interfaces Google Drive..."